packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -63,6 +63,46 @@ const defaultProps = {
   hideAccessoryDisplays: false,
 };
 
+const createConcpetDisplays = (compareConceptDisplays) => {
+  if (compareConceptDisplays.systolic && compareConceptDisplays.diastolic) {
+    if (compareConceptDisplays.systolic === compareConceptDisplays.diastolic) {
+      return <div className={cx('concept-display')}>{compareConceptDisplays.originalSystolic}</div>;
+    }
+    return (
+      <div className={cx('concept-display')}>
+        {compareConceptDisplays.originalSystolic}
+        {' / '}
+        {compareConceptDisplays.originalDiastolic}
+      </div>
+    );
+  }
+  if (compareConceptDisplays.systolic || compareConceptDisplays.diastolic) {
+    const conceptDisplayValue = compareConceptDisplays.originalSystolic || compareConceptDisplays.originalDiastolic;
+    return <div className={cx('concept-display')}>{conceptDisplayValue}</div>;
```

Comment:
This feels weird to me. If one was missing, why would we only show the one provided? How would a user differentiate this? Why not ` -- / value` or `value / --`?

Reply:
My understanding was it isn’t important to differentiate which one it applied to, you could either apply to both equally or shorthand the label/time by only applying to one if as an application you’ve already made sure they match and belong together, it’s only in cases where they mis-match that it would be important to show the differences. I can take this as a follow-up with clinical owners for verification.